aks/updateClusterTemplate.md

```diff
@@ -0,0 +1,143 @@
+<!--
+Copyright (c) 2021, Oracle and/or its affiliates.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+-->
+
+{% include variables.md %}
+
+# Post deployment actions for Oracle WebLogic Server cluster on {{ site.data.var.aksFullName }}
+
+This page documents how to update Oracle WebLogic cluster on {{ site.data.var.aksFullName }} with advanced configuration using Azure CLI.
+
+## Introduction
+
+{% include sub-template-advanced-usage.md %}
+
+While, there are some limitations:
+
+- No support to update a WebLogic cluster using older weblogic version, for example, you can not downgrade a 14.1.1.0 cluster to 12.2.1.4.
+
+- If you have enabled Azure Application Gatway Ingress Controller, you can not update the WebLogic cluster with TLS/SSL enabled to a cluster without TLS/SSL, otherwise, ingress will fail, you have to create new ingress for HTTP access.
+
+- You have to specify all required parameters, even though you are not going to update some of them.
+
+This document will guide you to update a WebLogic cluster using the advanced configurations.
+
+## Prerequisites
+
+{% include sub-template-prerequisites.md %}
+
+## Updating the existing WebLogic Server cluster
+
+The template will apply the new configuration in `parameters.json` to the running WebLogic cluster, please double check you have specified:
```

Comment:
double check you have specified ->  double check that you have specified

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb